lib/index.js

```diff
@@ -9,6 +9,8 @@ const path = require( 'path' );
 const createForkPool = require( './utils/createforkpool' );
 const logDisplay = require( './utils/displaylog' );
```

Comment:
The whole logger business needs some attention and we started improving it in https://github.com/cksource/mgit2/pull/46.